node/core/approval-voting/src/lib.rs

```diff
@@ -80,6 +83,8 @@ use crate::approval_db::v1::Config as DatabaseConfig;
 mod tests;
 
 const APPROVAL_SESSIONS: SessionIndex = 6;
+const APPROVAL_CHECKING_TIMEOUT: u64 = 2;
+const APPROVAL_CACHE_SIZE: usize = 128;
```

Comment:
We may want to raise this. Once we have 100 parachain cores we'll blow through this cache size in 1 or 2 blocks, and faster with forks